crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -1075,6 +1077,14 @@ mod tests {
         assert_eq!(json.ui, expected);
     }
 
+    #[test_case(r#"{ "daemon": true }"#, Some(true) ; "daemon_on")]
+    #[test_case(r#"{ "daemon": false }"#, Some(false) ; "daemon_off")]
+    #[test_case(r#"{}"#, None ; "missing")]
+    fn test_daemon(json: &str, expected: Option<bool>) {
```

Comment:
So something I missed: when we add new keys/fields to `turbo.json` we should always test both deserialization and serialization. When we don't we often create issues ([most recent issue caused by me forgetting to test this](https://github.com/vercel/turbo/issues/8311))

Reply:
sounds good! updated in https://github.com/vercel/turbo/pull/8728/commits/9065d956ad5d42abb1fd9744a98a2be105b01b9a